### ISSUE TITLE
Accuracy equals 0.903883 when iteration is 0. Why?

### ISSUE THREAD

**Issue by lismqi** [BVLC/caffe#5604]
### Issue summary
I began to train my model on my data. I found the accuracy was so high. I don't know what happened.The result maybe not persuasive.Here was the information....
-------------------------------------------------------------------------------------------------------
I0510 10:44:10.164682 32107 solver.cpp:337] Iteration 0, Testing net (#0)
I0510 10:44:17.416599 32107 solver.cpp:404]     Test net output #0: accuracy = 0.903883
I0510 10:44:17.416632 32107 solver.cpp:404]     Test net output #1: loss = 0.590191 (* 1 = 0.590191 loss)
--------------------------------------------------------------------------------------------------------
### Your system configuration
Operating system:14.04.1-Ubuntu
Python or MATLAB version (for pycaffe and matcaffe respectively):python2.7

**Comment by shaibagon** [comment]
Please use the [caffe-users list][1] for usage, installation, or modeling questions, or other requests for help.
You may also post questions on [stackoverflow][3], make sure you tag them with `caffe` tag.  
There is also [caffe.help][4] documenting the different layers of caffe.
Do not post such requests to Issues. Doing so interferes with the development of Caffe.

Please read the [guidelines for contributing][2] before submitting this issue.

 [1]: https://groups.google.com/forum/#!forum/caffe-users
 [2]: https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md
 [3]: http://stackoverflow.com/questions/tagged/caffe
 [4]: http://caffe.help/manual/layers.html
